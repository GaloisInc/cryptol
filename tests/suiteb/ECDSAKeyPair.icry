:m NISTCurves
:check
:l ECDSAKeyPair.cry
:check
