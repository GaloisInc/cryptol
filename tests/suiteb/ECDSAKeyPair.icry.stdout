Loading module Cryptol
Loading module Cryptol
Loading module PrimeEC
Loading module NISTCurves
property p192_valid_curve Using exhaustive testing.
Testing... Passed 1 tests.
Q.E.D.
property p224_valid_curve Using exhaustive testing.
Testing... Passed 1 tests.
Q.E.D.
property p256_valid_curve Using exhaustive testing.
Testing... Passed 1 tests.
Q.E.D.
property p384_valid_curve Using exhaustive testing.
Testing... Passed 1 tests.
Q.E.D.
property p521_valid_curve Using exhaustive testing.
Testing... Passed 1 tests.
Q.E.D.
Loading module Cryptol
Loading module PrimeEC
Loading module NISTCurves
Loading module Main
property p192_keypair_test Using exhaustive testing.
Testing... Passed 1 tests.
Q.E.D.
property p224_keypair_test Using exhaustive testing.
Testing... Passed 1 tests.
Q.E.D.
property p256_keypair_test Using exhaustive testing.
Testing... Passed 1 tests.
Q.E.D.
property p384_keypair_test Using exhaustive testing.
Testing... Passed 1 tests.
Q.E.D.
property p521_keypair_test Using exhaustive testing.
Testing... Passed 1 tests.
Q.E.D.
